assets/app/view/game/spreadsheet.rb

```diff
@@ -79,7 +81,13 @@ def or_history(corporations)
       end
 
       def render_history_titles(corporations)
-        or_history(corporations).map { |turn, round| h(:th, @game.or_description_short(turn, round)) }
+        conn = if @hide_connection_runs
```

Comment:
I'd consider the spreadsheet changes 'improvements' and should be done separate to the main PR as the PR is too large.

Reply:
The "spreadsheet changes" are just adding column to show the destination run

Reply:
yes, but it's another addition to a big PR.